src/store/slices/accounts/effects/addAccount.ts

```diff
@@ -0,0 +1,39 @@
+import { effect } from '../../../../react-vault';
+
+const getAccount = (spaceId: any, networkId: any, accountId: any, accountName: any) => {
+  return {
+    spaceId,
+    networkId,
+    accountId,
+    accountName,
+    importedAt: Date.now() + 1,
```

Comment:
No need to use +1 as we are adding only 1 account per moment and we won't have a situation when 2 accounts has the same time importedAt